src/driver/src/DefaultDriver.cpp

```diff
@@ -70,7 +70,7 @@ void reportMetrics(genny::metrics::Registry& metrics,
                    bool success,
                    metrics::clock::time_point startTime) {
     auto finishTime = metrics::clock::now();
-    auto actorSetup = metrics.operation(workloadName, "Setup", 0u);
+    auto actorSetup = metrics.operation("canary_" + workloadName, "Setup", 0u);
```

Comment:
Could you add a detailed comment here explaining why the `canary` prefix is necessary? Also link the perf trend chart regex for the `canary_` prefix.

Reply:
Sure